### PR TITLE
Cindy/image page ui merge

### PR DESCRIPTION
**Pull request by cindyxue**
Finish UI implementation for the home page. Did not add any logic and event listener for buttons.

http://localhost:8080/image-page.html

### CODE REVIEW

File: src/main/webapp/image-page.html [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Pupawrazzi</title>
+    <link rel="stylesheet" href="/css/main.css">
+    <link rel="stylesheet" href="/css/ImagePage.css">
+    <link href="https://fonts.googleapis.com/css?family=Crete+Round|Playfair+Display|Satisfy" rel="stylesheet">
+    <link rel="icon" href="images/doge.jpg">
+    <script src="/js/navigation-loader.js"></script>
+    <style>
```
Comment (minhphu20) [inline src/main/webapp/image-page.html]: Why do we have a separate CSS file and this embedded style file. Perhaps merge them? :)

**Review by minhphu20 — CHANGES_REQUESTED** [review]

File: src/main/webapp/css/ImagePage.css [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    margin-left: auto;
+    margin-right: auto;
+}
+
+/* Left button align left*/
+.alignleft {
+    float: left;
+    width: 15%;
+    background-color: Transparent;
+    background-repeat:no-repeat;
```
Comment (jchau9) [inline src/main/webapp/css/ImagePage.css]: background-repeat:no-repeat isn't necessary

File: src/main/webapp/css/ImagePage.css [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    cursor:pointer;
+    overflow: hidden; 
+}
+
+/* Right button align right*/
+.alignright {
+    float: right;
+    width: 15%;
+    background-color: Transparent;
+    background-repeat:no-repeat;
```
Comment (jchau9) [inline src/main/webapp/css/ImagePage.css]: background-repeat:no-repeat isn't necessary

File: src/main/webapp/css/ImagePage.css [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    border: none;
+    cursor:pointer;
```
Comment (jchau9) [inline src/main/webapp/css/ImagePage.css]: add space -> cursor: pointer;

File: src/main/webapp/css/ImagePage.css [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+/* Left button align left*/
+.alignleft {
+    float: left;
+    width: 15%;
+    background-color: Transparent;
+    background-repeat:no-repeat;
+    border: none;
+    cursor:pointer;
```
Comment (jchau9) [inline src/main/webapp/css/ImagePage.css]: add space -> cursor: pointer;

**Review by jchau9 — COMMENTED** [review]
Looks good!

**Pull request merged**